### ISSUE TITLE
内存占用溢出问题

### ISSUE THREAD

**Issue by xiaoxiaojiea** [wenet-e2e/wekws#154]
尊敬的开发者您好，首先感谢您们这项有价值的工作，但是在实际使用过程中我遇到了一些问题：训练过程中内存的占用在逐渐升高，直到最后训练被迫中止；我目前正在review源码检查问题，但是由于不是很熟悉代码，所以阅读的比较慢；同时在这里想了解一下您目前是否有解决方案，以解决当前的问题呢？

**Comment by mlxu995** [comment]
我对 fsmn 不太熟，@duj12 靖哥你看这里的 detach() 会影响 fsmn 的训练吗

**Comment by yang502** [comment]
如果是单卡训练的情况下，train_data_loader = DataLoader(train_dataset,
                                   batch_size=None,
                                   pin_memory=args.pin_memory,
                                   num_workers=args.num_workers)
                                   #prefetch_factor=args.prefetch)
